packages/webdriverio/tests/commands/element/saveScreenshot.test.js

```diff
@@ -37,21 +55,4 @@ describe('saveScreenshot', () => {
             elem.saveScreenshot('./file.txt')
         ).rejects.toEqual(expectedError)
     })
-
-    it('should fail if not existing directory', async () => {
```

Comment:
I think this test is still valueable because it ensure that no one accidentally remove the file path assertion.

Reply:
@christian-bromann there is a test for it already but in more simple way.
See lines 31-34 https://github.com/webdriverio/webdriverio/pull/3677/files#diff-a2fdb7dc0f85292704af730512ae75c7R33

we have verification that assertion is called with proper parameters

Reply:
ahh, yeah this is fine then 👍